bin/alluxio-mount.sh

```diff
@@ -130,43 +130,44 @@ function mac_hfs_provision_sectors() {
 function mount_ramfs_linux() {
   init_env $1
 
-  if [[ -z $ALLUXIO_RAM_FOLDER ]]; then
+  if [[ -z ${ALLUXIO_RAM_FOLDER} ]]; then
     ALLUXIO_RAM_FOLDER=/mnt/ramdisk
-    echo "ALLUXIO_RAM_FOLDER was not set. Using the default one: $ALLUXIO_RAM_FOLDER"
+    echo "ALLUXIO_RAM_FOLDER was not set. Using the default one: ${ALLUXIO_RAM_FOLDER}"
   fi
 
   mem_size_to_bytes
   TOTAL_MEM=$(($(cat /proc/meminfo | awk 'NR==1{print $2}') * 1024))
-  if [[ $TOTAL_MEM -lt $BYTE_SIZE ]]; then
-    echo "ERROR: Memory($TOTAL_MEM) is less than requested ramdisk size($BYTE_SIZE). Please reduce ALLUXIO_WORKER_MEMORY_SIZE"
+  if [[ ${TOTAL_MEM} -lt ${BYTE_SIZE} ]]; then
+    echo "ERROR: Memory(${TOTAL_MEM}) is less than requested ramdisk size(${BYTE_SIZE}). Please
+    reduce ALLUXIO_WORKER_MEMORY_SIZE"
     exit 1
   fi
 
-  F=$ALLUXIO_RAM_FOLDER
-  echo "Formatting RamFS: $F ($MEM_SIZE)"
-  if mount | grep $F > /dev/null; then
-    umount -f $F
+  F=${ALLUXIO_RAM_FOLDER}
+  echo "Formatting RamFS: ${F} (${MEM_SIZE})"
```

Comment:
I think either way works here. I'm fine with not brace-quote single char variables.

Per style guide:
"Don't brace-quote single character shell specials / positional parameters, unless strictly necessary or avoiding deep confusion. Prefer brace-quoting all other variables."

Updated, thanks!